skynet/modules/ttt/openai_api/app.py

```diff
@@ -38,6 +39,19 @@ def initialize():
         log.info(f'OpenAI API server started from {openai_api_server_path}')
 
 
+async def is_started():
+    # https://github.com/ggerganov/llama.cpp/blob/master/examples/server/README.md#get-health-returns-heath-check-result
+    try:
+        response = await http_client.get(f'http://localhost:{openai_api_server_port}/health')
```

Comment:
Don't we have the base URL in an env variable?

Reply:
for some reason i looked at that declaration and ignored its existence